tools/IncludeBin/IncludeBin.cpp

```diff
@@ -0,0 +1,33 @@
+#include <cstdio>
+#include <cstdlib>
+
+int main(int argc, char **argv) {
+  if (argc != 3) {
+    fprintf(stderr, "Usage: bin-include input_file output_file\n");
```

Comment:
nit: since we are in c++, probably should switch to std::cout/std::cerr etc.

Reply:
Is there any real advantage to the std:: streams for something tiny like this?  I admit I <3 printf format strings and hate looking up the formatting functions for streams so I always printf if I think I can get away with it O:-).